core/spec/lib/spree/migration_helpers_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe Spree::MigrationHelpers do
+  let(:helper) do
+    double.as_null_object.tap do |object|
+      object.extend(described_class)
+      object.extend(ActiveRecord::ConnectionAdapters::SchemaStatements)
+    end
+  end
+
+  subject { helper.safe_add_index(double, double) }
+
+  # regression test
+  describe "#safe_add_index" do
+    context "when the column exists" do
+      context "and the index does" do
+        it "is compatible with Ruby 3.0" do
+          expect { subject }.to_not raise_error(ArgumentError)
```

Comment:
RFC. I don't normally write specs with `to_not raise_error(...)` as be misleading if the test fails for other reasons, yet the spec passes. But, avoiding this, means a much more complicated test and setup.

Reply:
I think this is more than enough for something that we do no longer want to maintain. Thanks!